### ISSUE TITLE
Fewer privileged on pod for chown

### ISSUE THREAD

**Issue by leseb** [rook/rook#3504]
After https://github.com/rook/rook/pull/3498 pods running on OCP need `privileged` container. Let's see if we can allow CHOWN as a cap instead.

**Comment by djgalloway** [comment]
@leseb Am I seeing the following error in OCP 4.2 because of this issue?

```
Exec lifecycle hook ([chown --recursive --verbose ceph:ceph /var/log/ceph]) for Container "mgr" in Pod "rook-ceph-mgr-a-55fcf7d5d6-jjjbk_rook-ceph(f6bb560d-ae21-11e9-9ce2-00ccefa6f134)" failed - error: command 'chown --recursive --verbose ceph:ceph /var/log/ceph' exited with 1: chown: changing ownership of '/var/log/ceph': Permission denied
, message: "failed to change ownership of '/var/log/ceph' from ceph:ceph to ceph:ceph\nchown: changing ownership of '/var/log/ceph': Permission denied\n"
```

**Comment by travisn** [comment]
@djgalloway Can you check if you have the latest build? Looks like you're hitting #3499, which was fixed yesterday

**Comment by djgalloway** [comment]
Yep, this appears to be fixed in `rook-1.0.0-290.g72b9ec3a`.  Thanks @travisn!
